tests/src/test/java/org/wso2/micro/gateway/tests/common/CLIExecutor.java

```diff
@@ -56,7 +56,7 @@ public void generate(String label, String project, String security) throws Excep
                         .getPath()).getAbsolutePath();
         System.setProperty("user.dir", path.toString());
 
-        String[] initArgs = {"init", project};
+        String[] initArgs = {"init", project,"--etcd"};
```

Comment:
WE need to remov --etcd from the init command

Reply:
fixed with the following commit.
https://github.com/wso2/product-microgateway/pull/534/commits/7dbe875cf38a438b93b25af35edede7c112641aa

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/534/commits/6ded66c3dbd7dbd90e8e54c093bc11732f3efc15 .